packages/analytics/amplify_analytics_pinpoint_dart/test/queued_item_store_test.dart

```diff
@@ -0,0 +1,243 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/dart_queued_item_store.dart';
+
+import 'package:test/test.dart';
+
+void main() {
+  late DartQueuedItemStore db;
+
+  group('DartQueuedItemStore Tests', () {
+    setUpAll(() {
+      db = DartQueuedItemStore('test');
+    });
+
+    /// Reload db instance
+    Future<void> refreshDb() async {
+      await db.clear();
+    }
+
+    setUp(() async {
+      await refreshDb();
+    });
+
+    tearDown(() async {
+      await db.clear();
+    });
+
+    Future<void> validateStoredValues(
+      List<String> targetValues, {
+      int numToGet = 100,
+    }) async {
+      final readElements = await db.get(numToGet);
+
+      expect(readElements.length, targetValues.length);
+      for (var i = 0; i < readElements.length; i++) {
+        expect(
+          reason: 'Comparing element $i of readElements vs targetValues',
+          readElements.elementAt(i).value,
+          targetValues[i],
+        );
```

Comment:
It looks like this assertion expects the orders of elements in these lists are the same. if so can just use 
```dart
expect(targetValues, equals(readElements));
```
?